.travis.yml

```diff
@@ -36,6 +36,14 @@ jobs:
         - DEPLOY="true"
         - USE_MAVEN_REPO="true"
 
+    # sevntu (openjdk11)
+    - jdk: openjdk11
+      env:
+        - DESC="sevntu"
+        - CMD="mvn -e --no-transfer-progress clean verify -Pno-validations,sevntu"
```

Comment:
If we want precise execution
Better to use exact plugin execution, try  https://github.com/checkstyle/checkstyle/wiki/How-to-run-certain-phases-and-validations#how-to-generate-checkstyle-report

Reply:
please read https://github.com/checkstyle/checkstyle/pull/12346#discussion_r1042257603

Reply:
better to remove "verify" and execute ant maven task only.

Reply:
@romani So you are aware, the profile still exists so it still needs to be part of the command executed.

If that is ok, this is Done in the next push.

Reply:
Yes it is better expected to have profile